src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelDocManagementService.java

```diff
@@ -49,15 +49,16 @@ public class ExcelDocManagementService {
 
     public void uploadExcelDocument(String userToken, MultipleDetails multipleDetails, byte[] excelBytes) {
         MultipleData multipleData = multipleDetails.getCaseData();
-        log.info("Multiple Name is: " + multipleData.getMultipleName() + "for multiple reference: "
-                + multipleData.getMultipleReference());
+        log.info("Multiple Name is: {}for multiple reference: {}",
```

Comment:
I know the spacing isn't right in the original, but can we separate "{}for" -> "{} for" please?